app/renderer/js/utils/request-util.ts

```diff
@@ -38,11 +43,13 @@ class RequestUtil {
 		const certificate = CertificateUtil.getCertificate(
 			encodeURIComponent(domain)
 		);
+		const certificateFile = path.join(`${app.getPath('userData')}/certificates`, certificate);
```

Comment:
I reviewed this again, and to solve the issue with older versions you pointed out to me, I think we should just replace this line with the following:

```ts
        let certificateFile = null;
		if (certificate.includes('/')) {
			// certificate saved using old app version
			certificateFile = certificate;
		} else {
			certificateFile = path.join(`${app.getPath('userData')}/certificates`, certificate);
		}
```

Reply:
This works well for me on my end. @muskankhedia it'll be great if you can also try this out. Add the certificate to the app on `master`, checkout to this PR, then disconnect the concerned org and try adding it back. You should not get a prompt asking for you to confirm going ahead with adding the org.